packages/terra-search-field/src/SearchField.jsx

```diff
@@ -284,34 +297,44 @@ class SearchField extends React.Component {
       : undefined;
 
     return (
-      <div {...customProps} className={searchFieldClassNames}>
-        <div className={cx('input-group')}>
-          <input
-            {...additionalInputAttributes}
-            className={inputClass}
-            type="search"
-            placeholder={placeholder}
-            onChange={this.handleTextChange}
-            disabled={isDisabled}
-            aria-label={inputText}
-            aria-disabled={isDisabled}
-            onKeyDown={this.handleKeyDown}
-            onInput={this.handleInput}
-            ref={this.setInputRef}
-          />
-          {clearButton}
+      <div className={cx('search-container')}>
+        {isLabelVisible
+        && (
+        <label className={cx('label')}>{groupName}</label>
+        )}
+        <div className={cx('search-container')} role="search" aria-label="Search">
+          <div className={cx('search-container')} role="group" aria-label={groupName}>
```

Comment:
It seems incorrect for a child div to have the same class name as its parent container.  Also, do we really need both of these containers?

Reply:
Yes we need both the role divs per requirements and preferences on the Jira. And I tested that not having a display:inline styling for the child div makes the content display in a new line, which I am trying to avoid. 
Screenshot with and without the child div styling:
![image](https://github.com/cerner/terra-core/assets/19804033/16c21feb-3e6a-43bd-abca-60c522566a55)

![image](https://github.com/cerner/terra-core/assets/19804033/93b807c1-6dd5-47e9-8f69-da0cb72ec51f)

Reply:
I am not talking about the styling.  You are referencing a class name of search container for 3 separate containers.  That seems incorrect.

Reply:
What is the requirement specifically that forces both divs to be necessary?  How should it read to the end user.  On VoiceOver, I hear both search and group.  Is that expected?

Reply:
About having two divs, I am referring to the requirements mentioned on [UXPLATFORM-8109 ](https://jira2.cerner.com/browse/UXPLATFORM-8109)'s description under `Additional info`.

 I can look at combining one of the divs with existing ones if that doesn't modify the way the component renders.